samtranslator/model/function_policies.py

```diff
@@ -1,205 +0,0 @@
-from enum import Enum
```

Comment:
This must've been created since it served a purpose. Any idea what it was/is for?

Reply:
also curious about this, we need to be careful in case we have dynamically loaded some modules.

Reply:
I just checked it myself, no one is using it.